includes/forum-reactions.php

```diff
@@ -84,7 +84,7 @@ public function reaction_callback($data) {
                 $this->load_reactions($post_object->parent_id);
 
                 // Change reaction.
-                $response['status'] = $this->reaction_change($data['post_id'], get_current_user_id(), $data['reaction']);
+                $response['status'] = $this->reaction_change($data['post_id'], get_current_user_id(), $data['reaction'], $this->asgarosforum->get_post_author($post_object->id));
```

Comment:
We can use the `$post_object->author_id` variable here as it is used a few lines below as well.

Reply:
OK